tmk_core/protocol/usb_descriptor.h

```diff
@@ -278,7 +278,7 @@ enum usb_endpoints {
 #define KEYBOARD_EPSIZE 8
 #define SHARED_EPSIZE 32
 #define MOUSE_EPSIZE 8
-#define RAW_EPSIZE 32
+#define RAW_EPSIZE 64
```

Comment:
```suggestion
#define RAW_EPSIZE 32
```

Reply:
Whoops must have changed this when trying to test something, fixed 👍🏼